### DISCUSSION TITLE
Uncommended comment?

### DISCUSSION THREAD

**Discussion by Psycheros** [EDCD/EDDI#2297] [EDDI Standalone Q&A] [answered]
I'm not familiar enough with the project to feel comfortable to submit an issue with this but I believe in EDDI/EDDI.cs line 403 should be commented it reads "BUG".
404 returns true before 407 returns EDDI.Instance.gameIsBeta.

**Comment by Tkael** [comment, marked as answer]
This is a conditional statement for the compiler. When compiling a DEBUG build, it always returns true. When compiling a different kind of build (like a RELEASE build), it does not always return true and instead applies the formula.
